assets/i18n/strings_bn.i18n.json

```diff
@@ -0,0 +1,263 @@
+      "media": "মিডিয়া",
+      "text": "টেক্সট"
+    },
+    "shareIntentInfo": "আরো সহজে ফাইলসমূহ সিলেক্ট করার জন্য আপনি আপনার মোবাইলের \"Share\" ফিচারটি ব্যবহার করতে পারেন",
+    "nearbyDevices": "আশেপাশের ডিভাইসসমূহ",
+    "thisDevice": "এই ডিভাইস",
+    "scan": "ডিভাইসসমূহ সন্ধান করুন",
+    "help": "অনুগ্রহ করে নিশ্চিত করুন যে অন্য ডিভাসটিও একই ওয়াইফাই নেটওয়ার্কে রয়েছে।",
+    "placeItems": "শেয়ার করার জন্য ফাইল রাখুন।"
+  },
+  "settingsTab": {
+    "title": "সেটিংস",
+    "general": {
+      "title": "জেনারেল",
+      "theme": "থিম",
+      "themeOptions": {
+        "system": "সিস্টেম",
+        "dark": "ডার্ক",
+        "light": "লাইট"
+      },
+      "language": "ভাষা",
+      "languageOptions": {
+        "system": "সিস্টেম"
+      },
+      "minimizeToTray": "প্রস্থান করুন: ট্রেতে মিনিমাইজ করুন",
+      "launchAtStartup": "লগইন করার পরে অটোস্টার্ট করুন",
+      "launchMinimized": "অটোস্টার্ট: হিডেন থেকে শুরু করুন"
+    },
+    "receive": {
+      "title": "গ্রহণ করুন",
+      "quickSave": "@:general.quickSave",
+      "destination": "গন্তব্য",
+      "saveToGallery": "গ্যালারিতে মিডিয়া সেভ করুন"
+    },
+    "network": {
+      "title": "নেটওয়ার্ক",
+      "needRestart": "সেটিংস প্রয়োগ করতে সার্ভার রিস্টার্ট করুন!",
+      "server": "সার্ভার",
+      "alias": "উপনাম",
+      "port": "পোর্ট",
+      "portWarning": "আপনি একটি কাস্টম পোর্ট ব্যবহার করছেন তাই আপনি অন্য ডিভাইস থেকে এই ডিভাইকে খুঁজে নাও হতে পারে. (default: {defaultPort})",
+      "encryption": "এনক্রিপশন"
+    }
+  },
+  "selectedFilesPage": {
+    "deleteAll": "সব ডিলিট করুন"
+  },
+  "receivePage": {
+    "subTitle": {
+      "one": "আপনাকে একটি ফাইল পাঠাতে চায়।",
+      "other": "আপনাকে {n}টি ফাইল পাঠাতে চায়।"
+    },
+    "subTitleMessage": "আপনাকে একটি বার্তা পাঠিয়েছে:",
+    "subTitleLink": "আপনাকে একটি লিঙ্ক পাঠিয়েছে:",
+    "canceled": "সেন্ডার রিকুয়েস্টটি কেন্সেল করেছে।"
+  },
+  "receiveOptionsPage": {
+    "title": "অপশনসমূহ",
+    "destination": "@:settingsTab.receive.destination",
+    "appDirectory": "(@:appName folder)"
+  },
+  "sendPage": {
+    "waiting": "রেস্পন্সের অপেক্ষায় ...",
+    "rejected": "প্রাপক অনুরোধ প্রত্যাখ্যান করেছেন।"
+  },
+  "progressPage": {
+    "titleSending": "ফাইল পাঠানো হচ্ছে",
+    "titleReceiving": "ফাইল গ্রহণ করা হচ্ছে",
+    "savedToGallery": "Photos এ সেভ করা হয়েছে",
+    "total": {
+      "title": {
+        "sending": "মোট অগ্রগতি ({time})",
+        "finishedError": "ত্রুটি সহ সমাপ্ত হয়েছে",
+        "canceledSender": "সেন্ডারের দ্বারা বাতিল করা হয়েছে",
+        "canceledReceiver": "রিসিভার দ্বারা বাতিল করা হয়েছে"
+      },
+      "count": "ফাইলসমূহ: {curr} / {n}",
+      "size": "সাইজ: {curr} / {n}",
+      "speed": "স্পীড: {speed}/s"
+    }
+  },
+  "aboutPage": {
+    "title": "LocalSend সম্পর্কে"
+  },
+  "changelogPage": {
+    "title": "Changelog"
+  },
+  "aliasGenerator": {
+    "@info": "বিভিন্ন ভাষায় বিভিন্ন শব্দ থাকতে পারে, এটি 1:1 এর মতো নাও মিলতে পারে",
+    "adjectives": [
```

Comment:
Can the adjectives and fruits be combinated without modification? In some languages the adjective is dependent on the subjective (fruit in this case).

Secondly, how does the combination look like? Adjective + Fruit or Fruit + Adjective? You can lookup the latest English translation. There is a "combination" entry below the fruits.

Reply:
Extremely sorry for being late. I had some schoolwork.

Yes, the adjectives and fruits can be combined without modification and translation is not required also in that case. 

Adjective+fruit is okay.

please review the updated file and let me know if I can improve more.